src/commcare_cloud/ansible/roles/nginx/tasks/restart.yml

```diff
@@ -6,9 +6,9 @@
   check_mode: no
   failed_when: "result.rc != 0"
 
-- name: restart the nginx service
+- name: reload the nginx service
   become: yes
-  service: name=nginx state=restarted enabled=yes
+  service: name=nginx state=reloaded enabled=yes
```

Comment:
I'm pretty sure this is always what we want to do—that restart causes brief downtime without giving us much if anything. If any objections, I can remove this or make it its own PR

Reply:
Should the name of this file be changed to reload too?

Reply:
done